jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -1548,6 +1548,21 @@ void java_string_library_preprocesst::initialize_conversion_table()
   cprover_equivalent_to_java_string_returning_function
     ["java::org.cprover.CProverString.substring:(Ljava/Lang/"
      "StringBuffer;II)Ljava/lang/String;"] = ID_cprover_string_substring_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(Z)Ljava/lang/String;"] =
+      ID_cprover_string_of_bool_func;
```

Comment:
:pick: actually this could be implemented directly as a model, there is not complicated logic involved, so I don't know if this is needed :lollipop:

Reply:
Yes, I think in that case we can remove both `ID_cprover_string_of_bool_func` as well as `CProverString.toString(char)` and use `b ? "true" : "false"` directly.